### PR TITLE
feat: collect judge email addresses (FPLA-1664)

### PR DESCRIPTION
**Pull request by muyinatech**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-1664

### Change description ###
Add email address field to Allocated Judge screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java [hunk 1]
```diff
@@ -42,6 +42,7 @@ private static JudgeAndLegalAdvisor migrateJudgeAndLegalAdvisor(JudgeAndLegalAdv
             .otherTitle(allocatedJudge.getOtherTitle())
             .judgeLastName(allocatedJudge.getJudgeLastName())
             .judgeFullName(allocatedJudge.getJudgeFullName())
+            .judgeEmailAddress(allocatedJudge.getJudgeEmailAddress())
```
Comment (Jamesnelson117) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java:45]: When we migrate judge objects in create order, notice of proceedings etc the judge object associated with those events does not look like it captures judgeEmailAddress. It looks like this does not break anything (I see you've got a green build) but we might loose emailAddress in these cases. Is that a problem? Can you check wether this is the case?

Comment (muyinatech) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java:45]: I will check thanks.

Comment (muyinatech) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java:45]: As discussed, I will remove the migration of the judge's email address. As for create order, notice of proceedings etc, only a label for the allocated judge is retained. If there is a later requirement to use the judge's email address on another event other than Allocated Judge, it should be handle then.

Comment (muyinatech) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java]: Done.

File: ccd-definition/ComplexTypes/1_JudgeAndLegalAdvisor.json [hunk 2]
```diff
@@ -70,5 +70,14 @@
...
     "FieldType": "Text",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeEmailAddress",
+    "FieldType": "Email",
+    "ElementLabel": "Email Address",
+    "FieldShowCondition": "judgeTitle!=\"\"",
```
Comment (Jamesnelson117) [inline ccd-definition/ComplexTypes/1_JudgeAndLegalAdvisor.json:80]: Is this needed? Should this not be shown at all times?

Comment (muyinatech) [reply ccd-definition/ComplexTypes/1_JudgeAndLegalAdvisor.json:80]: When you first land on the screen for the 1st time none of the radio buttons are selected. The name & email fields are hidden. Upon selection these fields are shown.

**Pull request merged**